### PR TITLE
Dark mode added for hub.litmuschaos.io as requested by issue #1484

### PR DESCRIPTION
**Pull request by arkajyotiMukherjee**
Please review and test the dark mode features.

### CODE REVIEW

File: src/components/HomeHeader.js [hunk 1]
```diff
@@ -1,184 +1,200 @@
...
+                options={this.props.versions}
+                onChange={this.changeVersion}
+                value={getVersion(this.props.versions)}
+              />
+
+              <div
+                className="top-header-theme-changer"
+                onClick={() => this.props.toggleTheme()}
+              >
+                {this.props.isDarkTheme ? <FaMoon size={25} color='#FFFFFF' /> : <FaSun size={25} color='#FFD600' />}
```
Comment (is-ashish) [inline src/components/HomeHeader.js:163]: Please Add, Tooltip text, on mouse hovering "Dark Mode" and "Light Mode".

File: src/components/HomeHeader.js [hunk 2]
```diff
@@ -1,184 +1,200 @@
...
+                    alt="organization logo"
+                    src={process.env.PUBLIC_URL + "/icons/litmus.png"}
+                    className="logo-img"
                   />
                 </div>
               </a>
-              <Link to={process.env.PUBLIC_URL} className='home'>
-                <h3 className='top-header-home'>{<FaHome />}</h3>
+              <Link to={process.env.PUBLIC_URL} className="home">
+                <h3 className="top-header-home">{<FaHome />}</h3>
```
Comment (is-ashish) [inline src/components/HomeHeader.js:124]: Please, resize home icon make it some large

File: src/App.js [hunk 3]
```diff
@@ -1,35 +1,44 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { Redirect, Route, Switch } from 'react-router-dom';
-import { withRouter } from 'react-router';
+import React from "react";
+import PropTypes from "prop-types";
+import { Redirect, Route, Switch } from "react-router-dom";
```
[the remark this answers is not in the document]
Comment (arkajyotiMukherjee) [reply src/App.js:3]: This is forgot to delete, will do it.

File: src/components/ChartDetails.js [hunk 4]
```diff
@@ -1,43 +1,53 @@
...
 
-const ReactMarkdown = require('react-markdown');
-const customStyles = {
+const ReactMarkdown = require("react-markdown");
+
+const customStyles = (isDarkTheme) => ({
+  overlay: {
+    backgroundColor: isDarkTheme
+      ? "rgba(0,0,0,0.75)"
+      : "rgba(255,255,255,0.75)",
```
[the remark this answers is not in the document]
Comment (arkajyotiMukherjee) [reply src/components/ChartDetails.js:19]: Sure will make it hex colors

File: src/components/HomeHeader.js (hunk 1 above)
Comment (arkajyotiMukherjee) [reply src/components/HomeHeader.js:163]: Ok will add tooltips

File: src/components/HomeHeader.js (hunk 2 above)
Comment (arkajyotiMukherjee) [reply src/components/HomeHeader.js:124]: Ok

File: src/App.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (arkajyotiMukherjee) [reply src/App.js:3]: I misinterpreted the meaning sorry. Yeah, App.css was an empty file created by create-react-app template so was logo.svg which was the react logo which the dev forgot to remove so I removed those.
I replaced App.css with App.scss to provide some app styling.

[the remark this answers is not in the document]
Comment (arkajyotiMukherjee) [reply src/App.js:3]: The single quotes removal were recommended by ESLint.

File: src/App.js [hunk 5]
```diff
@@ -1,35 +1,44 @@
...
+          </Switch>
+        </div>
+      </div>
     );
   }
 }
 
 App.propTypes = {
-  location: PropTypes.object.isRequired
+  location: PropTypes.object.isRequired,
```
[the remark this answers is not in the document]
Comment (arkajyotiMukherjee) [reply src/App.js:37]: ES Lint reccomendation.

File: src/components/ChartDetails.js [hunk 6]
```diff
@@ -46,14 +56,14 @@ class UsesExplanation extends React.Component {
...
-export class ChartDetails extends React.Component {
+class ChartDetails extends React.Component {
   constructor(props) {
     super();
     this.state = {
       showModal: false,
       showVideoModal: false,
       isCollapsed: props.isCollapsed,
-      charts: props.charts
+      charts: props.charts,
```
[the remark this answers is not in the document]
Comment (arkajyotiMukherjee) [reply src/components/ChartDetails.js:66]: ES Lint reccomendation.

File: src/components/ChartDetails.js [hunk 7]
```diff
@@ -70,7 +80,7 @@ export class ChartDetails extends React.Component {
       this.setState({ isCollapsed: true });
     }
   };
-  createLink = listOfLink => {
+  createLink = (listOfLink) => {
```
[the remark this answers is not in the document]
Comment (arkajyotiMukherjee) [reply src/components/ChartDetails.js:83]: ES Lint reccomendation.

File: src/components/ChartDetails.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (arkajyotiMukherjee) [reply src/components/ChartDetails.js:19]: Done

File: src/components/HomeHeader.js (hunk 1 above)
Comment (arkajyotiMukherjee) [reply src/components/HomeHeader.js:163]: Done.

File: src/components/HomeHeader.js (hunk 2 above)
Comment (arkajyotiMukherjee) [reply src/components/HomeHeader.js:124]: Done. Submitting new PR and adding some screenshots.

**Comment by arkajyotiMukherjee** [comment]
# Screenshots for Theme Changing

Screenshot from 2020-05-14 23-34-29
Screenshot from 2020-05-14 23-34-36
Screenshot from 2020-05-14 23-35-46



Screenshot from 2020-05-14 23-34-04
Screenshot from 2020-05-14 23-34-07
Screenshot from 2020-05-14 23-34-11
Screenshot from 2020-05-14 23-34-23



Screenshot from 2020-05-14 23-33-52
Screenshot from 2020-05-14 23-33-58

**Comment by arkajyotiMukherjee** [comment]
### Updated the background colour in dark mode to a lighter shade to match the header image and changed the box-shadow properties in dark and light mode, as requested. Please review and let me know if you have any more suggestions.

Screenshot from 2020-05-16 23-08-49
Screenshot from 2020-05-16 23-12-17

**Comment by rajdas98** [comment]
Can you disable the eslint and push the older changes?
If we use the eslint, it required lot of changes

**Comment by arkajyotiMukherjee** [comment]
This PR is no more required as the website changed completely. I'm closing this.

**Pull request closed without merging**